lib/extend.js

```diff
@@ -11,6 +11,8 @@
  * and limitations under the License.
  */
 
+require('json5/lib/register');
```

Comment:
Dang. That is all we need to do?!?

Reply:
https://www.npmjs.com/package/json5#nodejs-require-json5-files

> Node.js require() JSON5 files
> When using Node.js, you can require() JSON5 files by adding the following statement.

`require('json5/lib/register')`
> Then you can load a JSON5 file with a Node.js require() statement. For example:

`const config = require('./config.json5')`